docs/api/cypress-api/catalog-of-events.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 title: Catalog of Events
+sidebar_position: 10
```

Comment:
I would expect events to be its own document / api navigation section. I would not expect to find these under the API section.

Reply:
I spent some time playing around with the layout of this section, and did a little bit of reorganization.

![image](https://user-images.githubusercontent.com/3003404/224107377-70a61829-c842-4460-9c25-d5d255638fab.png)

This based on the fact that I really like the way this looks in the table of contents:

![image](https://user-images.githubusercontent.com/3003404/224104225-bd0c8720-6bd4-4481-9055-90bf0ac4d16c.png)

I think it flows very naturally like this.